src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -229,13 +189,9 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 
 	logger.LogInformation("Looking for App-Insights loader generated by Oryx and export to NODE_OPTIONS if needed")
 	if shouldApplicationInsightsBeConfigured() {
-		loaderFile := filepath.Join(gen.SourcePath, consts.NodeAppInsightsLoaderFileName)
-
-		if !common.FileExists(loaderFile) {
-			createApplicationInsightsLoaderFile(loaderFile)
-		}
+		loaderFile := filepath.Join(gen.GlobalModulesPath, consts.NodeAppInsightsLoaderFileName)
```

Comment:
I think it's better to use the constant instead of using globalmodulespath in scriptgenerator structure